#! /bin/sh -e
## 30_tload_no_optargs.dpatch by Craig Small <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Removed unneeded optarg and optind variables form tload.c

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argum
ent"
                exit 1;;
esac

exit 0
@DPATCH@
--- procps-3.2.0.orig/tload.c
+++ procps-3.2.0/tload.c
@@ -30,9 +30,6 @@
 static int dly=5;
 static jmp_buf jb;
 
-extern int optind;
-extern char *optarg;
-
 static void alrm(int signo)
 {
     (void)signo;
